### ISSUE TITLE
[Feature Request] Android adaptive icon support

### ISSUE THREAD

**Issue by UseAfterFreee** [futo-org/grayjay-android#106]
Currently Grayjay does not have an adaptive icon, meaning that it does not look consistent on my homescreen where all app icons have the same color scheme and cosistent look.

See this for documentation on adaptive icons:
https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

**Comment by electro-1984** [comment]
Yeah, monochrome icons are awesome I wish all apps supported them OOTB
